pkg/asset/machines/aws/zones.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+// AvailabilityZones retrieves a list of availability zones for the given region
```

Comment:
nit: this is missing a trailing period ([docs][1]).

[1]: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences